projects/ng-rule-editor/src/lib/rule-editor.service.spec.ts

```diff
@@ -2,15 +2,54 @@ import { TestBed } from '@angular/core/testing';
 
 import { RuleEditorService } from './rule-editor.service';
 
+import {phq9ScoreLinkId, fhir } from '../../../../src/app/mock-data';
+
+const outputTotalScore = {
+  "url": "http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire-calculatedExpression",
+  "valueExpression": {
+    "description": "Total score calculation",
+    "language": "text/fhirpath",
+    "expression": "iif(%any_questions_answered, iif(%a.exists(), %a, 0) + iif(%b.exists(), %b, 0) + iif(%c.exists(), %c, 0) + iif(%d.exists(), %d, 0) + iif(%e.exists(), %e, 0) + iif(%f.exists(), %f, 0) + iif(%g.exists(), %g, 0) + iif(%h.exists(), %h, 0) + iif(%i.exists(), %i, 0), {})"
+  }
+};
+
+const outputItem = {
+  "url": "http://hl7.org/fhir/StructureDefinition/variable",
+  "valueExpression": {
+    "name": "i",
+    "language": "text/fhirpath",
+    "expression": "%questionnaire.item.where(linkId = '/44260-8').answerOption.where(valueCoding.code=%resource.item.where(linkId = '/44260-8').answer.valueCoding.code).extension.where(url='http://hl7.org/fhir/StructureDefinition/ordinalValue').value"
+  }
+};
+
+// @ts-ignore
```

Comment:
Can you also remove these 3 lines, they're not used.

Reply:
Fixed